### DISCUSSION TITLE
torch.cuda.OutOfMemoryError error (please help)

### DISCUSSION THREAD

**Discussion by ghost** [oobabooga/text-generation-webui#2298] [Q&A]
While starting the webui I got this error:
torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 32.00 MiB (GPU 0; 8.00 GiB total capacity; 7.30 GiB already allocated; 0 bytes free; 7.31 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF
Is it fixable?

**Comment by BetaDoggo** [comment]
It seems like you're out of video memory. The largest model you should be able to load with 8GB of vram is 7B in 4bit. There's a guide for using 4bit models on the [wiki](https://github.com/oobabooga/text-generation-webui/blob/main/docs/GPTQ-models-(4-bit-mode).md).

**Comment by danieljose** [comment]
Check this: [low vram guide](https://github.com/oobabooga/text-generation-webui/blob/main/docs/Low-VRAM-guide.md)
